src/components/Select/styles.css

```diff
@@ -0,0 +1,115 @@
+@import "@radix-ui/colors/black-alpha.css";
+@import "@radix-ui/colors/mauve.css";
+@import "@radix-ui/colors/violet.css";
```

Comment:
I have concerns about these styles, not sure if it is necessary to import these styles on component level. For example, another may also want to import them, so we'll end up with duplicates. Additionally these styles change global. I think it is better to move them to sdk global style.